source/hpc/research/research_plot.html

```diff
@@ -60,7 +60,8 @@
           ['Physics',90],
           ['NYU Shanghai',6],
           ['Social Science',3],
-          ['Psychology',2]
+          ['Psychology',2],
+          ['CRC',1]
```

Comment:
I think this should be 5?
Don't we need also to update years number of publications by the CRC publications?